var/spack/repos/builtin/packages/timemory/package.py

```diff
@@ -40,185 +66,307 @@ class Timemory(CMakePackage):
+    )
+    variant(
+        'use_arch',
+        default=False,
+        description=(
+            'Build all of timemory w/ cpu_target architecture '
+            'flags (default: roofline toolkit only)'
+        ),
+    )
+    variant(
+        'tls_model',
+        default='global-dynamic',
+        description='Thread-local static model',
+        multi=False,
+        values=('global-dynamic', 'local-dynamic', 'initial-exec', 'local-exec'),
+    )
+    variant('lto', default=False, description='Build with link-time optimization')
+    variant(
+        'statistics',
+        default=True,
+        description=('Build components w/ support for statistics ' '(min/max/stddev)'),
+    )
+    variant(
+        'extra_optimizations',
+        default=True,
+        description='Build timemory with extra optimization flags',
+    )
+    variant(
+        'cxxstd',
+        default='14',
+        description='C++ language standard',
+        values=('14', '17', '20'),
+        multi=False,
+    )
+    variant(
+        'cudastd',
+        default='14',
+        description='CUDA language standard',
+        values=('14', '17'),
+        multi=False,
+    )
+    variant(
+        'unity_build',
+        default=True,
+        description='Build with CMAKE_UNITY_BUILD=ON for faster builds '
+        'but larger memory consumption',
+    )
+    variant(
+        'mpip_library',
+        default=False,
+        description='Build stand-alone timemory-mpip GOTCHA library',
+    )
     variant('ompt', default=False, description=('Enable OpenMP tools support'))
-    variant('ompt_standalone', default=False,
-            description=('Enable OpenMP tools support via drop-in '
-                         'replacement of libomp/libgomp/libiomp5'))
-    variant('ompt_llvm', default=False,
-            description='Enable OpenMP tools support as part of llvm build')
-    variant('ompt_library', default=False,
-            description='Build stand-alone timemory-ompt library')
-    variant('allinea_map', default=False,
-            description='Enable Allinea ARM-MAP support')
-    variant('require_packages', default=True,
-            description=('find_package(...) resulting in NOTFOUND '
-                         'generates error'))
-
-    depends_on('cmake@3.11:', type='build')
+    variant(
+        'ompt_library',
+        default=False,
+        description='Build stand-alone timemory-ompt library',
+    )
+    variant('allinea_map', default=False, description='Enable Allinea ARM-MAP support')
+    variant(
+        'require_packages',
+        default=True,
+        description=('find_package(...) resulting in NOTFOUND ' 'generates error'),
+    )
+    variant(
+        'compiler', default=True, description='Enable compiler instrumentation support'
+    )
+    variant(
+        'ert',
+        default=True,
+        description='Enable extern templates for empirical roofline toolkit (ERT)',
+    )
 
     extends('python', when='+python')
```

Comment:
I'm thinking that now that this inherits from `PythonPackage`, `timemory~python` will not properly add to environments (since `PythonPackage` redefines how this works and expects that the package always extends Python).

I don't think that's a problem with this PR (rather it should be handled by Spack core developers to support this) but FYI.